cmd/validate_image.go

```diff
@@ -195,7 +194,12 @@ Write output in YAML format to stdout and in HACBS format to a file
 			if len(data.outputFile) > 0 {
 				data.output = append(data.output, fmt.Sprintf("%s=%s", applicationsnapshot.JSON, data.outputFile))
 			}
-			report := applicationsnapshot.NewReport(components)
+
+			publicKeyPEM, err := data.policy.PublicKeyPEM()
+			if err != nil {
+				return err
+			}
+			report := applicationsnapshot.NewReport(components, string(publicKeyPEM))
```

Comment:
Ack. It's fine for the public key, because that's the same thing for all components (at least for now anyways). We can make it more flexible in the future if needed.